### ISSUE TITLE
Color contrast does not meet minimum requirement - Siteimprove - Vertical/horizontal toolbar orientation link

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa801#9503]
From https://github.com/uiowa/uiowa/issues/9462

# Problem

Color contrast does not meet minimum requirement - WCAG 2.1 AA 1.4.3 Contrast (Minimum)

The only one that is not meeting the minimum is the vertical/horizontal orientation. Not sure if it was missed as part of https://www.drupal.org/project/drupal/issues/3097907

.toolbar-icon-toggle-vertical

<img width="136" height="87" alt="Image" src="https://github.com/user-attachments/assets/a45d01ce-1e22-42e9-ae87-a950c2341288" />
<img width="281" height="101" alt="Image" src="https://github.com/user-attachments/assets/efe58d52-42f7-4844-bcdc-7df6da11bb2d" />

**Comment by briand44** [comment]
Reopening this to discuss as Siteimprove browser plugin is still reporting this while authenticated.

**Comment by joewhitsitt** [comment]
It is a scoping issue. The fix was applied to docroot/profiles/custom/sitenow/css/admin-overrides.css which isn't loaded for an authenticated public path (e.g. https://sandbox.prod.drupal.uiowa.edu/), only the backend admin theme (e.g. https://sandbox.prod.drupal.uiowa.edu/admin/structure).

**Comment by briand44** [comment]
## Grooming
 
**Quadrant:** 

Clear

 **Discussion:**
 - https://github.com/uiowa/uiowa/issues/9503#issuecomment-4237978882
   - still an issue while authenticated on `https://sandbox.prod.drupal.uiowa.edu/`
   - resolved while authenticated to `https://sandbox.prod.drupal.uiowa.edu/admin/content`
 - adjust css scoping to apply to both authenticated admin paths as well as authenticated public paths

**Proposed solution or definition of done:**
- adjust css to fix color contrast issue on all authenticated pages
- verify Siteimprove is no longer reporting
